src/explore-education-statistics-admin/src/pages/release/content/components/__tests__/EditableKeyStatDataBlock.test.tsx

```diff
@@ -370,6 +395,53 @@ describe('EditableKeyStatDataBlock', () => {
       });
     });
 
+    test('submitting edit form calls `onSubmit` handler with trimmed updated guidance title', async () => {
+      tableBuilderService.getDataBlockTableData.mockResolvedValue(
+        testTableDataResponse,
+      );
+
+      const onSubmit = jest.fn();
+
+      render(
+        <EditableKeyStatDataBlock
+          releaseId="release-1"
+          keyStat={keyStatDataBlock}
+          isEditing
+          onSubmit={onSubmit}
+        />,
+      );
+
+      await waitFor(() => {
+        expect(screen.getByText(/Edit/)).toBeInTheDocument();
+      });
+
+      userEvent.click(
+        screen.getByRole('button', {
+          name: 'Edit key statistic: DataBlock indicator',
+        }),
+      );
+
+      await waitFor(() => {
+        expect(screen.getByLabelText('Trend')).toBeInTheDocument();
+      });
+
+      userEvent.clear(screen.getByLabelText('Guidance title'));
+      await userEvent.type(
+        screen.getByLabelText('Guidance title'),
+        '  New guidance title  ',
+      );
+
+      userEvent.click(screen.getByRole('button', { name: 'Save' }));
+
+      await waitFor(() => {
+        expect(onSubmit).toHaveBeenCalledWith<[KeyStatDataBlockFormValues]>({
```

Comment:
Wrong style for typing arrays here. You'll struggle to find this syntax used Typescript codebases & this it goes against our current code styles

```
KeyStatDataBlockFormValues[]
```

Reply:
Changed similar cases in the same file and in `EditableKeyStatText.test.tsx`